llvm/lib/Support/APFloat.cpp

```diff
@@ -1007,6 +1023,10 @@ IEEEFloat &IEEEFloat::operator=(IEEEFloat &&rhs) {
 }
 
 bool IEEEFloat::isDenormal() const {
+  // The E8M0 format does not support denormals.
+  if (semantics == &semFloat8E8M0FN)
```

Comment:
Also would expect this to just work out

Reply:
Yes, you are right, Matt! This does not need any special handling.
Removed this check in the latest revision.